src/backend/effects/queues/effect-queue-manager.js

```diff
@@ -101,7 +101,7 @@ class EffectQueueManager extends JsonDbManager {
         }
 
         item.queue = effectQueueRunner.getQueue(itemId);
-
+        item.length = item.queue.length;
```

Comment:
adding this so the express api also has easy access to this value. Its present when calling for all queues but not individual queues so assume just a small oversight.

Reply:
When I implemented that originally, the paradigm was "fetch all queues, get length. Fetch specific queue, get the array of items instead for more advanced use"

the intention there was that if you needed the length, you'd get it from the array as the array is already present.

Reply:
Completely get it, I'm adding it for the less capable tools that can fetch against the api but only take the json key and don't permit any type of count/logic capabilities.